source/client-side-encryption/client-side-encryption.rst

```diff
@@ -914,6 +912,17 @@ Drivers MUST support a BSON document option named ``encryptedFields`` for any
 will be interpreted by the helper method and MUST be passed to the `create`_
 command.
 
+.. note::
+   Users are not expected to set the ``escCollection`` and ``ecocCollection`` in
+   ``encryptedFields``. SERVER-74069 added server-side validation for those fields
+   and no longer allows names to deviate from the following:
+
+   - ``enxcol_.<collectionName>.esc``
+   - ``enxcol_.<collectionName>.ecoc`
+
+   Drivers SHOULD NOT document the ``escCollection`` and ``ecocCollection``
```

Comment:
Any reason not to use `MUST NOT` here? Once this spec change is implemented, the options will serve no purpose.

Reply:
No reason. Updated to use `MUST NOT`.